### ISSUE TITLE
the accuracy on the OCID_grasp dataset and the problem of the stack

### ISSUE THREAD

**Issue by MJ0108** [stefan-ainetter/grasp_det_seg_cnn#6]
Hello! thanks for your excellent work!
I have some questions:
1.when I run the test part, I only got the visible results—sets of images with grasp candidates and segmentation results. How can I get the grasp accuracy and seg . IoU? I can't find them.
2.can you tell me what the operation of the stack is in the grasp refinement head? 
Thank you very much!

**Comment by stefan-ainetter** [comment]
Hi,
really sorry for the late reply!
About 1): The evaluation script is currently not part of the repository. We plan to add it in the next days.
About 2): The input for the MLP is a 4d tensor of shape (NxCxHxW), where W,H are width and height of the feature map. C is the dimension where we stack the two feature maps, and we use the [torch.cat()](https://pytorch.org/docs/stable/generated/torch.cat.html)
 operation to do so. We generate these feature map pairs N-times, where N defines the number of grasp candidates that we want to refine.

@emptybottle, the grasp refinement head is currently not part of this repository.

**Comment by MJ0108** [comment]
> Hi, really sorry for the late reply! About 1): The evaluation script is currently not part of the repository. We plan to add it in the next days. About 2): The input for the MLP is a 4d tensor of shape (NxCxHxW), where W,H are width and height of the feature map. C is the dimension where we stack the two feature maps, and we use the [torch.cat()](https://pytorch.org/docs/stable/generated/torch.cat.html) operation to do so. We generate these feature map pairs N-times, where N defines the number of grasp candidates that we want to refine.
> 
> @emptybottle, the grasp refinement head is currently not part of this repository.

thank you!

**Comment by stefan-ainetter** [comment]
Evaluation script for calculating Jaccard index and segmentation IOU was added in commit https://github.com/stefan-ainetter/grasp_det_seg_cnn/commit/2492d5ec78f831c327e817246e822cdfce9e16ad.
